lib/loadtest.js

```diff
@@ -139,14 +139,29 @@ class Operation {
 	 */
 	startClients() {
 		const url = this.options.url;
+		const strBody = JSON.stringify(this.options.body);
```

Comment:
From #186 I read your comment on moving this `if(options.body)` That would cause the bug. What happens is, because the strBody is replaced after the first match, there will be no original string in the body left that would match the `indexParam` because of this, any body with index param will always have same strings. For example:
Say, I have `indexParam = changeThis` and my body looks something like this
```
{
   ...
   blah: changeThis
   ...
}
```
now all my requests will have same body with `blah:0` instead of being unique for every request.

Reply:
OK, so please take into account the other comments and we can merge!